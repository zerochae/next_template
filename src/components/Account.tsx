const Account = (props: { name: string; mbti: string }) => {
  return (
    <div>
      <h2>{props.name}</h2>
      <p>{props.mbti}</p>
    </div>
  );
};

export default Account;
