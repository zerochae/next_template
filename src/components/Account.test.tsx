import Account from "./Account";
import { render } from "@testing-library/react";

it("matches snapshot", () => {
  const utils = render(<Account name="gray" mbti="INFP" />);
  expect(utils.container).toMatchSnapshot();
});

it("shows the props correctly",()=>{
  const utils = render(<Account name="gray" mbti="INFP"/>)
  utils.getByText("gray")
  utils.getByText("INFP")
})