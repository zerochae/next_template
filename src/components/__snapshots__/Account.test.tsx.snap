// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <div>
    <h2>
      gray
    </h2>
    <p>
      INFP
    </p>
  </div>
</div>
`;
